src/test/resources/application-test.yml

```diff
@@ -15,4 +15,10 @@ api:
     url: http://will-be-replaced-by-wireMockServer-bean
 
 dynamodb:
-  print-details-table-name: print-details
\ No newline at end of file
+  print-details-table-name: print-details
+  scheduler-lock-table-name: scheduler-locks
+
+jobs:
+    print-requests:
+        name: BatchPrintRequests
+        cron: 0 0 */1 * * *
```

Comment:
The cron you have put here is "at midnight, every day of the month" (according to crontab.guru)
What does this mean for our integration tests in general?

Your `@Scheduled` class at the moment does nothing, which is fine. I guess all you are doing here is providing a value so that Spring is happy and starts the context.

What happens when the `@Scheduled` class has some behaviour in it? Could this potentially get triggered when we didn't expect it to? (eg: maybe someone is working really late over midnight? (it has been known! 🙄 ). Or more likely, what happens if our CI stack is running the tests over a midnight boundary?) Might this be a problem somehow?

But on the subject of integration tests generally (and I know it wasn't in your spike, so thats fine), have you had any thoughts as to how we might test `@Scheduled` class once we have some behaviour in it?
Clearly we will write the class in such a way that we can unit test it with mocks etc. But will we need to integration test it? Will we need to test it in such a way that the cron is triggered, and it queries the database, and produces a batch, and uploads it to the SFTP server etc etc ?
That might be really really really hard to do 🤔 

I honestly dont know the answer to that, but I wonder if it might be a case that we (somehow) disable scheduled jobs altogether (all of them) in our integration test context, so that they dont 'randomly' trigger etc. And that we just accept that we cant integration test this part of the system in this way?

What do you think? Would definitely be worth getting other views and opinions on this from the rest of the API dev team 🤔